components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/CodeIDTokenTokenResponseValidator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.common;
+
+/**
+ * Validator for hybrid flow code id_token token requests
+ */
+public class CodeIDTokenTokenResponseValidator extends CodeTokenResponseValidator {
```

Comment:
If this class doesn't have an implementation why do we need this ? can't we use the same class as . CodeTokenResponseValidator ?

Reply:
Yes, we can use it. But won't there be any customization that might be needed in the future?